toggl/__init__.py

```diff
@@ -1,11 +1,7 @@
-from pbr.version import VersionInfo
```

Comment:
Sooo I have missed these changes. Basically removing the `pbr` here. 

I am not against it, but to my knowledge, this is half way done change. For example, there needs to be automation that updates the `__version__` upon release... `pbr` uses git tags on build and includes that information during distribution build directly into the distro package... That was one of the reasons why I went with it. So for now I am reverting the `pbr` removal, but if you feel like doing it over again with completing it then feel free ;-)

Reply:
... Believe me or not, but I completely don't recall these changes. I think when I rebased your branch, I must have messed it up somehow and these popped back in. Apologies!